src/app/server/Server.cpp

```diff
@@ -93,6 +93,10 @@ CHIP_ERROR Server::Init(AppDelegate * delegate, uint16_t secureServicePort, uint
     err = mFabrics.Init(&mServerStorage);
     SuccessOrExit(err);
 
+    err = mGroupsProvider.Init();
```

Comment:
Add a comment that since this depends on storage, we should initialize after storage init.

Reply:
Commend added.